src/components/Header/Header.tsx

```diff
@@ -0,0 +1,21 @@
+import {Link} from "react-router-dom";
+import header_logo from '../../images/header_logo.jpg';
+import {Button, Input} from 'antd';
+import css from './header.module.css';
+
+const {Search} = Input
+
+export const Header = () => {
+    return (
+        <div className={css.container}>
+            <Link to=''>
+                <div className={css.logo}>
+                    <img src={header_logo} alt={'logo'} style={{width: '100%', height: '100%'}}/>
```

Comment:
имена переменных даем в камел кейсе, стилизуем только с помощью css
```suggestion
                    <img src={headerLogo} alt={'logo'} className={css.logoImage} />
```